volume/drivers/nfs/nfs.go

```diff
@@ -211,14 +212,18 @@ func (d *driver) Create(
 	source *api.Source,
 	spec *api.VolumeSpec) (string, error) {
 
-	volumeID := locator.Name
-	if volumeID == "" && source.Parent != "" {
-		volumeID = d.getNewSnapVolID(source.Parent)
-		dlog.Infof("Creating snap vol id: %s", volumeID)
+	if locator.Name == "" {
```

Comment:
This works, but instead use the style `if len(location.Name) == 0` to check for empty strings.

Reply:
@lpabon okay